digital-twin/service/src/main/java/com/microsoft/azure/sdk/iot/digitaltwin/service/DigitalTwinServiceAsyncClient.java

```diff
@@ -56,24 +56,24 @@
      * @return The observable to the json string specifying the updated state  of the digital twin representation
      * @throws IOException Throws IOException if the json deserialization fails
      */
-    Observable<String> updateDigitalTwinProperties(@NonNull String digitalTwinId, @NonNull final String interfaceInstanceName, @NonNull String propertyPatch) throws IOException;
+    Observable<String> updateDigitalTwinProperties(@NonNull String digitalTwinId, @NonNull final String componentName, @NonNull String propertyPatch) throws IOException;
 
     /**
      * Invoke a digital twin command on the given interface instance that is implemented by the given digital twin
```

Comment:
interface instance => component

elsewhere too

Reply:
@abhipsaMisra  is going to carry on to fix naming for service client. There are more in generated code and payload might be changed as well in future. Service side is not ready yet.